src/main/java/com/github/wovnio/wovnjava/Headers.java

```diff
@@ -129,74 +139,34 @@ String langCode() {
         }
     }
 
+    /*
+     * Take as input a location string of any form (relative path, absolute path, absolute URL).
+     * If the location needs a Wovn language code, return an absolute URL string of that location
+     * with language code of the current request language. Else return the location as-is.
+     *
+     * Do not modify `location` if
```

Comment:
Personally I think this code is pretty self explanatory and reads almost the same as the comment, so I would consider it redundant.

Reply:
Fair enough 👍